server/deployment/src/main/java/io/quarkiverse/openapi/server/generator/deployment/codegen/ApicurioCodegenWrapper.java

```diff
@@ -0,0 +1,129 @@
+package io.quarkiverse.openapi.server.generator.deployment.codegen;
+
+import static io.quarkiverse.openapi.server.generator.deployment.CodegenConfig.getBasePackagePropertyName;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Path;
+import java.util.Enumeration;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import org.apache.commons.io.IOUtils;
+import org.eclipse.microprofile.config.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.apicurio.hub.api.codegen.JaxRsProjectSettings;
+import io.apicurio.hub.api.codegen.OpenApi2JaxRs;
+import io.quarkus.bootstrap.prebuild.CodeGenException;
+
+public class ApicurioCodegenWrapper {
+
+    private static final Logger log = LoggerFactory.getLogger(ApicurioOpenApiServerCodegen.class);
```

Comment:
You're getting the wrong logger.

```suggestion
    private static final Logger log = LoggerFactory.getLogger(ApicurioCodegenWrapper.class);
```